File: Data Structure/20255/Solutions/Find_Maximum_Element_in_Array.js
class Solution{
  findMaximum(arr){
    if(!arr || arr.length === 0) return null;
    let max = arr[0];
    for(let i = 1; i < arr.length; i++){
      if(arr[i] > max) max = arr[i];
    }
    return max;
  }
}
function solve(input){
  const lines = input.trim().split("\n");
  const n = parseInt(lines[0]);
  const arr = lines[1].split(" ").map(Number);
  const sol = new Solution();
  const ans = sol.findMaximum(arr);
  if(ans === null || ans === undefined) console.log("Array is Empty");
  else console.log(ans);
}
const fs = require('fs');
solve(fs.readFileSync(0, 'utf-8'));

File: Data Structure/20255/Solutions/Find_Maximum_Element_in_Array.py
class Solution:
    def findMaximum(self, arr):
        if not arr or len(arr) == 0:
            return None
        max_val = arr[0]
        for num in arr:
            if num > max_val:
                max_val = num
        return max_val

def solve():
    n = int(input())
    arr = list(map(int, input().split()))
    sol = Solution()
    result = sol.findMaximum(arr)
    
    if result is None:
        print("Array is Empty")
    else:
        print(result)

if __name__ == '__main__':
    solve()

